src/main/resources/flyway/sql/V9__populate_institution_country_region.sql

```diff
@@ -0,0 +1,266 @@
+--Populate data for Institution, Country, and Region --
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+INSERT INTO institution (id, name)
+VALUES
+  (uuid_generate_v4(), 'OICR'),
+  (uuid_generate_v4(), 'FBI');
+
+INSERT INTO country (id, name)
+VALUES
```

Comment:
Are we not also validating the 2 character country codes in short names? Is this just done in the short_name validator and not matched from the DB?